tools/check-switch/testdata/src/a/a.go

```diff
@@ -0,0 +1,161 @@
+package a
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+)
+
+func switchOK(v interface{}) {
+	switch v := v.(type) {
+	case *types.Document:
+		fmt.Println(v)
+	case *types.Array:
+		fmt.Println(v)
+	case float64:
+		fmt.Println(v)
+	case string:
+		fmt.Println(v)
+	case types.Binary:
+		fmt.Println(v)
+	case types.ObjectID:
+		fmt.Println(v)
+	case bool:
+		fmt.Println(v)
+	case time.Time:
+		fmt.Println(v)
+	case types.NullType:
+		fmt.Println(v)
+	case types.Regex:
+		fmt.Println(v)
+	case int32:
```

Comment:
```suggestion
	case int32, int64:
```

How that case will be handled? Let's test it